elasticsearch/Chart.yaml

```diff
@@ -5,7 +5,7 @@ maintainers:
 - email: helm-charts@elastic.co
   name: Elastic
 name: elasticsearch
-version: 6.8.15
```

Comment:
Chart versions are only bumped when we have a new release (https://github.com/elastic/helm-charts/blob/master/CONTRIBUTING.md#releases)